test/e2e/apimachinery/resource_quota.go

```diff
@@ -43,7 +43,7 @@ import (
 
 const (
 	// how long to wait for a resource quota update to occur
-	resourceQuotaTimeout = 30 * time.Second
```

Comment:
only used in 
https://github.com/kubernetes/kubernetes/pull/98906/files#diff-e1a46008c4ddbb9f83858cb22ce56ca0f820b78142a1e8abbbb11aa59127a09bL1714 before